xbmc/dialogs/GUIDialogKeyboardGeneric.cpp

```diff
@@ -11,6 +11,7 @@
 #include "GUIDialogNumeric.h"
 #include "GUIUserMessages.h"
 #include "ServiceBroker.h"
+#include "dialogs/GUIDialogKaiToast.h"
```

Comment:
Include should be surrounded by `#ifdef TARGET_ANDROID`, because toasts are only needed on Android.

Reply:
fine I'll add the #ifdef